docs/source/release-notes/2021-1-0-Eos.md

```diff
@@ -0,0 +1,166 @@
+    1. Fix being unable to sit in the real world in VR without the avatar and viewpoint being popped up to a standing position (fixes https://github.com/vircadia/vircadia/issues/825). With the new option "Allow my avatar to stand", we can choose to have the avatar stand only when the user is standing. The default behavior is the same as before (always allowed to stand; allowed to lean only when user is standing), except that now, when foot trackers are used, the avatar is never made to stand artificially.
+    2. Fix very wrong walk/fly directions when using foot tracking or when using the mode that allows the avatar to sit when the user sits (previously "Avatar lean behavior: disabled").
+    3. Fix VR walk speeds being too slow at small avatar scale and too fast at large avatar scale (fixes https://github.com/vircadia/vircadia/issues/851). The avatar scale was being factored-in twice in the velocity calculation.
+    4. The "Avatar lean behavior" setting was unclear and had muddled logic. It's now split into two settings that work more usefully: "Allow my avatar to stand" and "Allow my avatar to lean".
+    5. The previous experimental mode that allowed the avatar to sit on the floor ("Avatar leaning behavior: disabled") is improved by now having the avatar follow the user's orientation ("Allow my avatar to lean: Always, no recenter").
+    6. Fix unreliable hand-relative walk/fly directions, due to avatar-frame vectors being wrongly transformed by the head rotation. UpdateMotors was transforming avatar-space directions by the head rotation (even if hand-relative walking was chosen). It could give the appearance of working, but only when the avatar rotation followed the head rotation.
+    7. Fix for the recentering not behaving as requested by the user after getting up from click-to-sit (always popping up to standing position).
+    8. Fix for incorrect vertical position of the avatar and viewpoint after changing recenter mode while not standing in the real world.
+    9. The logic controlling the direction of `MyAvatar`'s action motor was hard to follow and maintain, as it was split between two functions. Now it's clearer and all done in `calculateScaledDirection`.
+    10. Leaning and lean recentering now work as intended at any avatar scale.
+* API: Add `useOriginalPivot` property to model entities allowing entities to use their original pivot. (#951)
+* Graphics: Improve rendering of transparent entities. (#953)
+* API: Add `billboardMode` property for all entity types allowing entities to face the user as specified in the following modes: none, yaw, full. (#961)
+* App: The 'Appreciate' app has been added as a default app to Interface. (#974)
+* API: Add `userAgent` property to web entities allowing the web entity to use the specified user agent when browsing. (#998, #1017)
+* App: Allow any user to teleport to any user within a domain using the 'People' app. (#991)
```

Comment:
Is this right? Teleport permissions have been removed?

Reply:
Unless I am mistaken, there aren't any teleport permissions per se. Rather there was a limitation in that you had to be logged in order for the "teleport to user" function to work.

Reply:
No. It is supposed to abide to the permissions set by the user. The setting is still there in v2021.1.0-rc2:
![1Auswahl_846](https://user-images.githubusercontent.com/11144627/107611403-36544380-6c44-11eb-96b0-d97f33a3413e.png)